docs/index.md

```diff
@@ -0,0 +1,21 @@
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing external access to HTTP/HTTPS services in a
+Kubernetes cluster via a Gateway and an HTTPRoute resource. This requires the Kubernetes
+cluster in question to have an Gateway API controller already deployed into it.
+
+## Project and community
+
+gateway API is an open-source project that welcomes community contributions, suggestions, fixes and constructive feedback.
+
+* [Read our Code of Conduct](https://ubuntu.com/community/code-of-conduct)
+* [Join the Discourse forum](https://discourse.charmhub.io/tag/synapse)
+* [Discuss on the Matrix chat service](https://chat.charmhub.io/charmhub/channels/charm-dev)
+* Contribute and report bugs to [the Synapse operator](https://github.com/canonical/gateway-api-integrator-operator)
+* Check the [release notes](https://github.com/canonical/gateway-api-integrator-operator/releases)
+
+## Contributing to this documentation
+
+Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/) to enable easy collaboration. Please use the “Help us improve this documentation” links on each documentation page to either directly change something you see that’s wrong, or ask a question, or make a suggestion about a potential change via the comments section.
+
+If there’s a particular area of documentation that you’d like to see that’s missing, please [file a bug](https://github.com/canonical/gateway-api-integrator-operator/issues).
+
+# Contents
```

Comment:
Will more be added underneath "Contents"? I see that there aren't any files right now for tutorials, how-to's, etc., but I'm wondering whether that's on the to-do list for these docs. Otherwise might want to remove for now?

Reply:
I added more contents here, I also renamed this section to `in this documentation`